sensors/triggers/argo-workflow/argo-workflow.go

```diff
@@ -126,19 +116,14 @@ func (t *ArgoWorkflowTrigger) Execute(events map[string]*v1alpha1.Event, resourc
 
 	switch op {
 	case v1alpha1.Submit:
-		file, err := ioutil.TempFile("/bin/workflows", workflow.Name)
+		file, err := ioutil.TempFile("/bin/workflows", name)
```

Comment:
`file, err := ioutil.TempFile("", name)`

This will create the file under the default temp directory, then `/bin/workflows` is not required in Dockerfile.

See https://golang.org/pkg/io/ioutil/#TempFile